src/app/components/views/homepage/homepage.component.ts

```diff
@@ -1,29 +1,35 @@
-import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
 import { Subject, Subscription, take, takeUntil } from 'rxjs';
 
 import { JobData, JobsResponse } from 'src/app/models/jobs-response.model';
 import { WeatherAlertResponse } from 'src/app/models/weather-alert.model';
 import { JobService } from 'src/app/services/job.service';
 import { WeatherService } from 'src/app/services/weather.service';
 
+
 @Component({
   selector: 'app-homepage',
   templateUrl: './homepage.component.html',
   styleUrls: ['./homepage.component.scss']
 })
 export class HomepageComponent implements OnInit, OnDestroy {
+
+  @Input() safetyDocumentUpdated: boolean;
```

Comment:
I don't think this would traditionally be an Input, I think we would probably wire it up to a service file with some sort of observable. We can worry about that later and leave it as a loosey-goosey variable here in the homepage component.